src/quests/prologue/tasks/cmc.ts

```diff
@@ -0,0 +1,39 @@
+import { Task } from "grimoire-kolmafia";
+import { getWorkshed, myAdventures, runChoice, totalTurnsPlayed, use, visitUrl } from "kolmafia";
+import { $item, get, have } from "libram";
+
+import { CMC } from "../../../prefs/properties";
+
+export function CMCInstalled(): boolean {
```

Comment:
This function doesn't seem to be checking if the CMC is actually installed. Please reconsider the function name, or move around the function logic to better reflect this method signature.